app/src/main/java/de/xikolo/controllers/dialogs/HelpdeskTopicAdapter.kt

```diff
@@ -59,11 +71,21 @@ class HelpdeskTopicAdapter(private val onTopicClickedListener: OnTopicClickedLis
     }
 
     private fun bindTopicViewHolder(holder: CourseTitleViewHolder, position: Int) {
+
         val topic = contentList.get(position) as HelpdeskTopic
-        holder.topicTitle.text = topic.first
+        if (topic.first == TicketTopic.COURSE) {
+            val course = CourseDao.Unmanaged.find(topic.second)
+            holder.topicTitle.text = course?.title
+            holder.topicSlug.text = course?.slug
+            holder.topicYear.text = getCourseYear(course).toString()
+            holder.optionalViewGroup.visibility = View.VISIBLE
+        } else {
+            holder.topicTitle.text = topic.first.toString()
+            holder.optionalViewGroup.visibility = View.GONE
+        }
 
         holder.layout.setOnClickListener {
-            onTopicClickedListener?.onTopicClicked(topic.first, topic.second, topic.third)
+            onTopicClickedListener?.onTopicClicked(holder.topicTitle.text.toString(), topic.first, topic.second)
```

Comment:
Can we introduce a local variable for this first argument? I can only think of side effects when using the `TextView` getter.
Like
```
var t = if(topic.first == TicketTopic.COURSE){
    ...
    course?.title
} else {
    ...
    topic.first.toString()
}
holder.topicTitle.text = t
...
onTopicClickedListener(t,...)